torch/csrc/jit/script/init.cpp

```diff
@@ -476,6 +476,35 @@ void initJitScriptBindings(PyObject* module) {
             }
             return result;
           })
+      .def(
+          "save",
+          [](std::shared_ptr<Function> self,
+             const std::string& filename,
+             const ExtraFilesMap& _extra_files = ExtraFilesMap()) {
+            Module module;
+            // Make a graph with a fake self argument
+            auto graph = self->graph()->copy();
+            graph->insertInput(0, "self");
+            module.module_object()->type()->compilation_unit().create_function(
+                "forward", graph);
+            module.save(filename, _extra_files);
+          },
+          py::arg("filename"),
+          py::arg("_extra_files") = ExtraFilesMap())
+      .def(
+          "save_to_buffer",
+          [](std::shared_ptr<Function> self,
+             const ExtraFilesMap& _extra_files = ExtraFilesMap()) {
+            std::ostringstream buf;
+            Module module;
+            auto graph = self->graph()->copy();
```

Comment:
Can we put the implementation in Function::save, and avoid duplicating the logic between save and save_to_buffer?